test/test_helper.exs

```diff
@@ -53,4 +56,10 @@ defmodule RollbarAPI do
   def call(conn, _test) do
     send_resp(conn, 404, "Not Found")
   end
+
+  defp with_delay(fun, delay \\ 100) do
```

Comment:
I think we should match on `Cowboy.shutdown` return and fail if it anything else but `:ok`:
```elixir
:ok = Cowboy.shutdown(__MODULE__.HTTP)
:timer.sleep(100)
:ok
```
Please note that you patch changing when sleep is called.

And same in `start/1`.